CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.3.0 (2020-03-10)
+
+- Added the country field component. [#55](https://github.com/blackbaud/skyux-lookup/pull/55)
```

Comment:
Should we add a brief description about the component? Something like:

Added the country field component to select a country from search results.

If not, what you have is good!

Reply:
Doesn't matter to me. I just took what we had done for character count but we can add more if you want to :)

Reply:
I'm fine with leaving it as then :)